draft-ietf-quic-tls.md

```diff
@@ -1763,8 +1763,10 @@ those produced by the application protocol that QUIC serves.
 Note:
 
 : TLS session tickets and address validation tokens are used to carry QUIC
-  configuration information between connections.  These MUST NOT be used to
-  carry application semantics.  The potential for reuse of these tokens means
+  configuration information between connections.  Specifically, to enable a
+  server to efficiently recover state that is used in connection establishment
+  and address validation.  These MUST NOT be used to communicate application
+  semantics between endpoints.  The potential for reuse of these tokens means
```

Comment:
This actually seems pretty subtle. Instead, I would say:

These have no application-level semantics; clients MUST treat them as opaque values.